components/Table/columnDefinitions/playerHeroesColumns.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
-import { defaultSort, transformedSort } from './utility';
+import { defaultSort, transformedSort, useOriginalValueSort } from './utility';
 import { HOST_URL } from '../../../yasp.config';
 import styles from './playerHeroesColumns.css';
 import PiePercent from '../../PiePercent';
 
 const winPercentTransform = (list, field, property) => list[field][property] / list.games[property];
+const winPercentWithTransform = (list, field, property) => list[field][property] / list.with_games[property];
+const winPercentAgainstTransform = (list, field, property) => list[field][property] / list.against_games[property];
 
 const getPercentWin = (wins, games) => Math.ceil(1000 * (wins / games)) / 10;
+const getPercentWinWith = (with_win, with_games) => Math.ceil(1000 * (with_win / with_games)) / 10;
+const getPercentWinAgainst = (against_win, against_games) => Math.ceil(1000 * (against_win / against_games)) / 10;
 
```

Comment:
You can just use winPercentTransform and getPercentWin as they are doing the same things with different argument names